bootstrap/secret/secure.go

```diff
@@ -104,6 +107,10 @@ func (p *SecureProvider) GetSecret(path string, keys ...string) (map[string]stri
 	}
 
 	if err != nil {
+		if strings.Contains(err.Error(), PathNotFoundError) {
+			return nil, pkg.NewErrSecretStore(fmt.Sprintf("Could not retrieve path '%s'. %s", path, err.Error()))
+		}
+
```

Comment:
I think this check needed here. HasSecrets can check for `ErrPathNotFound` for `go-mod-secrets`